nautobot/dcim/tests/test_models.py

```diff
@@ -1264,6 +1264,21 @@ def test_child_devices_are_not_saved_when_unnecessary(self):
         self.assertNotEqual(child_mtime_after_parent_rack_update_save, child_mtime_after_parent_site_update_save)
 
 
+class DeviceBayTestCase(ModelTestCases.BaseModelTestCase):
```

Comment:
Looks like some of the generic tests are passing, likely because we don't in develop have a factory for DeviceBay instances. Might be easily addressable by adding a `setUp()` that creates some?